lib/livebook/runtime.ex

```diff
@@ -104,7 +104,12 @@ defprotocol Livebook.Runtime do
 
   Similar to `t:markdown/0`, but with no special markup.
   """
-  @type plain_text_output :: %{type: :plain_text, text: String.t(), chunk: boolean()}
+  @type plain_text_output :: %{
+          type: :plain_text,
+          text: String.t(),
+          chunk: boolean(),
+          style: keyword()
```

Comment:
I think of the Runtime protocol as the source of truth specification that the rest of Livebook is compatible with, so let's be precise here as well:

```suggestion
          style: [{:color | :font_weight | :font_size, String.Chars.t()}]
```